Dropdown/Dropdown.module.less

```diff
@@ -0,0 +1,39 @@
+// Dropdown.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.Dropdown {
+	.applySkins({
+		.item {
+			border-color: gray;
+			border-style: solid;
+			border-width: 1px;
+		}
+		.transitionContainer {
+			position: absolute;
+			display: inline-block;
+			z-index: 1;
+			pointer-events: none;
+			min-width: 312px;
+			width: 100%;
```

Comment:
Should we keep this to 100%? Or how should we support different dropdown widths?

Reply:
At this point we need keep the width,  don't have enough information for different sizes for dropdown.